main/integration-tests/__test__/api-tests/studies/list-studies.test.js

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('List study scenarios', () => {
```

Comment:
What do you think of starting a convention where we say 'get' instead of 'list' when naming the files and when describing the tests?  this is mainly because list is just 'get' on the collection resource

Reply:
For this, I personally would like to stick to the description of the API itself. Having multiple get-study test suites, or multiple test cases, hitting different API routes would be confusing to the reader at first.